pkg/console/operator/operator.go

```diff
@@ -273,5 +276,16 @@ func (c *consoleOperator) removeConsole(ctx context.Context, recorder events.Rec
 	_, _, updateConfigErr := resourceapply.ApplyConfigMap(ctx, c.configMapClient, recorder, configmap.EmptyPublicConfig())
 	errs = append(errs, updateConfigErr)
 
-	return utilerrors.FilterOut(utilerrors.NewAggregate(errs), errors.IsNotFound)
+	// filter out 404 errors, which indicate that resource is already deleted
+	err := utilerrors.FilterOut(utilerrors.NewAggregate(errs), apierrors.IsNotFound)
+
+	if err != nil {
+		statusHandler.AddCondition(consolestatus.HandleAvailable("", "ConsoleRemoved", err))
+		statusHandler.AddCondition(consolestatus.HandleDegraded("", "ConsoleRemoved", err))
+		return err
+	}
+
+	statusHandler.AddCondition(consolestatus.HandleAvailable("", "ConsoleRemoved", nil))
+	statusHandler.AddCondition(consolestatus.HandleDegraded("", "ConsoleRemoved", nil))
```

Comment:
Do we need to handle upgradeable as well?

Reply:
Was wondering about that TBH, we probably should.